src/auth/discord-auth.service.ts

```diff
@@ -78,4 +80,19 @@ export class DiscordAuthService implements IAuthProvider {
             },
         });
     }
+
+    async findUserByEmails(emails): Promise<any | null> {
```

Comment:
Typescript is complaining. 
```suggestion
    async findUserByEmails(emails: string): Promise<any | null> {
```